.changeset/fuzzy-feet-exist.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/frontend-plugin-api': patch
+'@backstage/plugin-app': patch
+---
+
+Deprecated the `namespace` option for `createExtensionBlueprint` and `createExtension`, these are no longer required and will default to the `pluginId` instead
```

Comment:
Thinking this might be a good place to through in a bit of extra guidance for how to override for a specific name space, i.e. migrating an existing namespace + `createExtensionOverrides` to a module

Reply:
It would be `createFrontendModule` now right?